### PR TITLE
app-admin/yadm: add 3.2.2

### PR DESCRIPTION
**Pull request by ixti**
cc: @gyakovlev

### CODE REVIEW

File: app-admin/yadm/yadm-3.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..10} )
```
Comment (juippis) [inline app-admin/yadm/yadm-3.2.2.ebuild:6]: Could you ping upstream about python-3.11 support? I tried and the tests fail, but don't see an upstream ticket about it. (Maybe it's fixed in -git already?)

File: app-admin/yadm/yadm-3.2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-shells/bash
+	app-crypt/gnupg
```
Comment (juippis) [inline app-admin/yadm/yadm-3.2.2.ebuild:22]: Sort.

**Review by juippis — APPROVED** [review]

File: app-admin/yadm/yadm-3.2.2.ebuild (hunk 1 above)
Comment (ixti) [reply app-admin/yadm/yadm-3.2.2.ebuild:6]: i will open a ticket with the upstream

File: app-admin/yadm/yadm-3.2.2.ebuild (hunk 2 above)
Comment (ixti) [reply app-admin/yadm/yadm-3.2.2.ebuild:22]: i see pr was merged. i can open a revbump with sorted list, or i can add a note to myself to sort it on next version bump

Comment (juippis) [reply app-admin/yadm/yadm-3.2.2.ebuild:22]: No need, 809e34279ce2af190a3b706ea2b9c4c77e72a7f2

Comment (ixti) [reply app-admin/yadm/yadm-3.2.2.ebuild:22]: oh. awesome. thank you!

Comment (juippis) [reply app-admin/yadm/yadm-3.2.2.ebuild:22]: No no, thank YOU! Especially if we can get python-3.11 support.

**Pull request closed without merging**